remotespark/datawidgets/ipythondisplay.py

```diff
@@ -0,0 +1,11 @@
+from IPython.core.display import display, HTML
+
+
+class IpythonDisplay(object):
```

Comment:
This should be a class if and only if IpythonDisplay is an instantiable/mockable object, but it doesn't look like it is.